src/smt/proof_post_processor.h

```diff
@@ -136,6 +121,13 @@ class ProofPostprocessCallback : public ProofNodeUpdaterCallback, protected EnvO
                     const std::vector<Node>& args,
                     CDProof* cdp,
                     Node res = Node::null());
+  /**
+   * Called when we require expanding a macro step from within the method above.
```

Comment:
you mean that this is called when one would recursively expand macros? Please clarify.

Reply:
Done, PTAL